libraries/AP_HAL_ChibiOS/hwdef/CUAV-Nora/hwdef.dat

```diff
@@ -21,7 +21,7 @@ STM32_ST_USE_TIMER 2
 FLASH_RESERVE_START_KB 128
 
 # order of UARTs (and USB)
-SERIAL_ORDER OTG1 USART2 USART6 USART1 UART4 UART8 UART7 OTG2
+SERIAL_ORDER OTG1 USART2 USART6 USART1 UART4 UART8 UART7 OTG2 USART3
```

Comment:
we have up till now always put OTG2 last in the serial order, so USART3 would be SERIAL7 and OTG2 SERIAL8
and you happy to do that? Or do you particularly want USART3 to be SERIAL8?

Reply:
Thanks for the reminder, I have adjusted.